tests/test_inference_client.py

```diff
@@ -197,6 +198,14 @@ def test_text_to_speech(self) -> None:
         audio = self.client.text_to_speech("Hello world")
         self.assertIsInstance(audio, bytes)
 
+    def test_zero_shot_image_classification(self) -> None:
+        output = self.client.zero_shot_image_classification(self.image_file, "tree,woman,cat")
```

Comment:
```suggestion
        output = self.client.zero_shot_image_classification(self.image_file, ["tree", "woman", "cat"])
```
must be a list of labels